ts/components/MessageComponent.tsx

```diff
@@ -22,46 +23,29 @@ export type Props = OwnProps;
  */
 class MessageComponent extends React.Component<Props> {
   public formatDate(date: string): string {
-    moment.locales();
-
-    const messageDate = moment(new Date(date)).format();
+    const distance = distanceInWordsToNow(new Date(date));
     if (
-      moment(messageDate, "YYYYMMDDh")
-        .fromNow()
-        .includes("hours ago") === true ||
-      moment(messageDate, "YYYYMMDDh")
-        .fromNow()
-        .includes("hour ago") === true ||
-      moment(messageDate, "YYYYMMDDh")
-        .fromNow()
-        .includes("minutes ago") === true ||
-      moment(messageDate, "YYYYMMDDh")
-        .fromNow()
-        .includes("minute ago") === true
-    ) {
-      return moment(messageDate).format("H.mm");
-    } else if (
-      moment(messageDate, "YYYYMMDDh")
-        .fromNow()
-        .includes("a day ago") === true
+      distance.includes("hours") === true ||
```

Comment:
wait, what is going on here? I don't understand this logic, what is the intent here? - @fpersico